### DISCUSSION TITLE
Package OpenCV.Net installed but dosen't appear on toolbox

### DISCUSSION THREAD

**Discussion by AstridNicolini15** [bonsai-rx/bonsai#1596] [Q&A]
Hello,

I have a bonsai workflow where i use the crop node on a video. I externalized the region of interest property in order to change the coordinates as i like but i can't find how to get input into it. It requires an OpenCV.Net.Rect element which exists in the OpenCV.Net library, but i can't find the node on my bonsai toolbox. I check on the package manager that OpenCV.Net is indeed installed, but no element of this package appear on my toolbox.  

Do you have any ideas where this problem can come from ?

Thank you

**Comment by bruno-f-cruz** [comment]
Right click `Crop` -> `CreatePropertySource` -> `RegionOfInterest`. If the constructor operator does not exist for a given property, this is the way to have access to one, when possible.

**Reply by AstridNicolini15** [reply]
Thank you for your answer ! 
I still don't understand how to modify the values of the region of interest source, without doing it manually in the property panel. Indeed I face the same issue of having to give a Net.Rect element as an input.

**Reply by bruno-f-cruz** [reply]
Not sure I follow, you asked how to define an input to the `RegionOfInterest`, wouldn't this allow you to do so? Input whatever values you want in Rect and feed it to the externalized property. Can you explain a bit better what you are trying to achieve? maybe include a workflow?

**Reply by AstridNicolini15** [reply]
Yes, I wasn't very clear. I can explain better. 
My overall goal is to automatically load property values when opening my bonsai workflow. Indeed I will run the same bonsai workflow several times with different individuals, hence some property values have to be personnalized but I do not want to change them manually every time. To do so, I use Bash (a command line interface) to assign values to externalized properties at the opening of the workflow. My issue is that I can't pass a Net.Rect element. Hence, I am looking for a way to pass ints and then format them to the proper Rect element (which is constituted of 4 ints).
